upgrade/test_crossgrade.sh

```diff
@@ -257,7 +257,9 @@ upgradeIstioAtVersionUsingIstioctl(){
   istioctl_path="${3}"/bin
   # shellcheck disable=SC2072
   if [[ "${TO_TAG}" > "1.5" ]]; then
-    "${istioctl_path}"/istioctl upgrade --skip-confirmation
+    # The following is a workaround for the test failure due to that the charts is
+    # no longer bundled in the release (https://github.com/istio/istio/issues/23172).
+    "${istioctl_path}"/istioctl manifest apply --skip-confirmation --charts "${3}"/manifests
```

Comment:
actually to run `make gen-charts` before the original upgrade command would also work. But yeah, to replace `upgrade` to `manifest apply` as a workaround seems fine to me.